modules/core/include/opencv2/core/cvstd.hpp

```diff
@@ -627,13 +627,15 @@ String::String(const char* s, size_t n)
     : cstr_(0), len_(0)
 {
     if (!n) return;
+    if (!s) return;
```

Comment:
Probably we should have an assertion here instead. We should not "eat" bad parameters.